src/tiden/dockermanager.py

```diff
@@ -473,9 +478,12 @@ def default(val, default_value):
         # if need to redefine entry command by your own
         command_str = ''
         if command:
-            command_str = command
-            if default(bash_commands, True):
-                command_str = f'bash -c "{command_str}"'
+            if isinstance(command, list) or isinstance(command, tuple):
+                command_str = ' '.join(command)
+            else:
+                command_str = command
```

Comment:
```suggestion
            command_str = command if isinstance(command, str) else ' '.join(command)
```

Because the types are strictly limited

Reply:
Oh, but this breaks if `command` is None

Reply:
yeah

Reply:
but we have `if` before this line

Reply:
fixed